string name
float64[] timestamps
string signal
